{"uuid": "63d1d88a-5c86-4205-803c-364074060b46", "children": ["97a849f9-eb51-4f04-b433-a524b1f3ee2a"], "befores": [{"name": "set_up", "status": "passed", "start": 1653324320850, "stop": 1653324324293}], "afters": [{"name": "set_up::0", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python310\\lib\\site-packages\\allure_commons\\_allure.py\", line 200, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python310\\lib\\site-packages\\_pytest\\fixtures.py\", line 899, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\User\\OOP_Project\\Base\\base.py\", line 26, in set_up\n    self.driver.quit()\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python310\\lib\\site-packages\\selenium\\webdriver\\chromium\\webdriver.py\", line 240, in quit\n    self.service.stop()\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python310\\lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 147, in stop\n    self.send_remote_shutdown_command()\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python310\\lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 128, in send_remote_shutdown_command\n    if not self.is_connectable():\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python310\\lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 116, in is_connectable\n    return utils.is_connectable(self.port)\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python310\\lib\\site-packages\\selenium\\webdriver\\common\\utils.py\", line 107, in is_connectable\n    socket_ = socket.create_connection((host, port), 1)\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python310\\lib\\socket.py\", line 833, in create_connection\n    sock.connect(sa)\n"}, "start": 1653324324495, "stop": 1653324325648}], "start": 1653324320850, "stop": 1653324325648}